dts/arm/nxp/pinmux_nxp_kinetis.h

```diff
@@ -23,10 +26,14 @@
 #define K_PORT_MuxAlt14				14
 #define K_PORT_MuxAlt15				15
 
-#define DT_NXP_PINMUX_DEFINE(k_name, k_port, k_pin, k_mux) \
-	k_name##_##pt##k_port##k_pin: k_name##_##pt##k_port##k_pin { \
-		pin = <k_pin>; \
-		mux = <k_mux>; \
-	}
+#define DT_PINCFG_FLAG(flag) flag;
+#define DT_PINCFG_FLAGS(...) \
+       MACRO_MAP_CAT(DT_PINCFG_FLAG __VA_OPT__(,) __VA_ARGS__)
+
+#define DT_NXP_PINMUX_DEFINE(k_name, k_port, k_pin, k_mux, ...) \
+        k_name##_##pt##k_port##k_pin: k_name##_##pt##k_port##k_pin { \
+               nxp,kinetis-pins = < &port##k_port k_pin k_mux >; \
+               DT_PINCFG_FLAGS(__VA_ARGS__) \
```

Comment:
What is this for?

Reply:
Its a hold over from non-generated dtsi files.

Reply:
Will remove in next iteration.